sdk/cosmos/azure-cosmos/azure/cosmos/aio/database.py

```diff
@@ -358,31 +371,31 @@ def list_containers(
 
     @distributed_trace
     def query_containers(
-        self,
-        query=None,  # type: Optional[str]
-        parameters=None,  # type: Optional[List[Dict[str, Any]]]
-        max_item_count=None,  # type: Optional[int]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> AsyncItemPaged[Dict[str, Any]]
+            self,
+            query: Union[str, Dict[str, Any]],
```

Comment:
This was previously optional?

Reply:
changed back and packaged into kwargs since it's optional